src/Features/VisualBasic/Portable/Structure/Providers/WhileBlockStructureProvider.vb

```diff
@@ -13,7 +13,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
         Inherits AbstractSyntaxNodeStructureProvider(Of WhileBlockSyntax)
 
         Protected Overrides Sub CollectBlockSpans(node As WhileBlockSyntax,
-                                                  spans As ArrayBuilder(Of BlockSpan),
+                                                  ByRef spans As TemporaryArray(Of BlockSpan),
```

Comment:
so, tbh, `block structure` is theplace i would *not* expect to see wins.  in other words, we're almost always going to be +4 elemetns rights?

Reply:
I'm guessing it's going to be perf-neutral for this case, but I like it because it's hit often and helps exercise the non-copyable analyzer we're relying on.

Reply:
sgtm.